### DISCUSSION TITLE
Hello  All, I am now completing my second project but 2nd one is based on tkinter designer

### DISCUSSION THREAD

**Discussion by HarshscGithub** [ParthJadhav/Tkinter-Designer#158] [Show and tell]
Hello  All, I am now working on my second project but 2nd one is based on tkinter designer. I am 13 year old. Thanks.

**Comment by HarshscGithub** [comment]
If any one want to see then let me know.

**Reply by ParthJadhav** [reply]
That's awesome! Share the project here.

**Reply by HarshscGithub** [reply]
Shared : https://github.com/HarshscGithub/Atarbals-Modern-Antivirus

**Comment by HarshscGithub** [comment]
### Today I Released working Beta of my second project so please **_use it as an example_**. And you can also give me feedback on my Atarbals Antivirus and Ki-Lo antivirus please.

**Comment by HarshscGithub** [comment]
hello it is now created

**Comment by douloveppt** [comment]
牛逼呀，有操作的呀这个人！

**Reply by HarshscGithub** [reply]
Type in English if talking about me.

**Reply by HarshscGithub** [reply]
"Niubi, there is an operation, this person!"  -> Translation didn't understood.

**Reply by douloveppt** [reply]
> "Niubi, there is an operation, this person!" -> Translation didn't understood.

Hahaha, you are so cute😜
Niubi means great

**Reply by HarshscGithub** [reply]
Just wanted to ask will you develop my project I can't give you anything in exchange. But I can give credit equally
